phpBB/common.php

```diff
@@ -52,7 +52,7 @@
 	}
 
 	// $phpbb_root_path accounts for redirects from e.g. /adm
-	$script_path = trim(dirname($script_name)) . '/' . $phpbb_root_path . 'install/index.' . $phpEx;
+	$script_path = trim(dirname($script_name)) . '/' . $phpbb_root_path . 'install/app.' . $phpEx;
```

Comment:
This is inside an `if (!defined('PHPBB_INSTALLED'))` block. And btw it wouldn't be that bad to move the whole ACP into controllers in this PR ;) so obsolete?